### PR TITLE
Task 5 added

### PR DESCRIPTION
**Pull request by Jhon112**
add models/engine/file_storage.py, models/engine/__init__.py
Create FileStorage class with attributes and methods requested.
Edit models/init to instance FileStorage and have only one instance that gets and sets objs from file.json
I also edited to_dict method from BaseModel. 

> A copy was needed because we were not able to use storage.save()
> This was due to __dict__ being edited by adding it __class__ key,
> Also we were editing the value of updated_at and created_at keys
> so when we were going to dump the obj it has a datetime obj as
> value on updated_at key

### CODE REVIEW

**Comment by monicajaimesc** [comment]
We have to check out this together, working in task 6 and fixing one bug in task 4. I won't accept the git merge as we talked ;)

**Comment by Jhon112** [comment]
This version is now working as asked, it's creating the objects now and they are being printed as BaseModel objs and not as dictionaries, we'll review this code together to show what was wrong

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -3,60 +3,60 @@
...
+	JSON reader and writer.
+
+	Basically it will store the json to a file and deserialize
+	that json to an obj, this way we restore all the data
+	of the program. JSON files work as our database
+	"""
+
+	__file_path = 'file.json'
+	__objects = {}
+
```
Comment (monicajaimesc) [inline models/engine/file_storage.py:22]: Ok, cool.. I saw the same mistake

**Review by monicajaimesc — APPROVED** [review]
Cool, gonna try it in my consola and then we can talk about it. ;)

**Pull request merged**